packages/utils/src/npm.ts

```diff
@@ -1,77 +1,38 @@
 import * as os from "os";
 import process from "process";
-import fs from "fs";
 import RegClient from "@qiwi/npm-registry-client";
-import { resolve as resolveUrl } from "url";
 import { joinPaths } from "./fs";
 import { Logger } from "./logging";
-import { createTgz } from "./io";
-
-export const npmRegistryHostName = "registry.npmjs.org";
-export const npmRegistry = `https://${npmRegistryHostName}/`;
 
 export const cacheDir = joinPaths(process.env.GITHUB_ACTIONS ? joinPaths(__dirname, "../../..") : os.tmpdir(), "cache");
 
-type NeedToFixNpmRegistryClientTypings = any;
-
 export class NpmPublishClient {
```

Comment:
this only tags and deprecates now. I'd rename to NpmTagger or something
and we seem to mostly create it on-demand, so maybe callers should switch to calling `tag`/`deprecate` directly.